code/src/components/SubmitButton.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import forwardButton from '../assets/forward.png'
+import { initializeApp } from "firebase/app";
+import { getDatabase, ref, set } from 'firebase/database'
+
+// Firebase Configuration
+const firebaseConfig = {
+    apiKey: "AIzaSyDGbArzFfRKxXoM7v1E9jhVQqS7HfXYtlk",
+    authDomain: "project-survey-week10.firebaseapp.com",
+    databaseURL: 'https://project-survey-week10-default-rtdb.europe-west1.firebasedatabase.app/',
+    projectId: "project-survey-week10",
+    storageBucket: "project-survey-week10.appspot.com",
+    messagingSenderId: "245167263996",
+    appId: "1:245167263996:web:ade40b556cd8a503b00779",
+    measurementId: "G-EKP1BVXVN8"
+  };
```

Comment:
Okay you need to demo this backend-database-thing some time! Teach me master

Reply:
This was my sneaky part of the project! I'm happy you noticed - I saved everyones replies to a database, so now I have little model cards on everyones personalities on a separate secret website 😇